pkg/loadbalancer/maps/lbmaps.go

```diff
@@ -762,6 +804,10 @@ func (f *FaultyLBMaps) DumpMaglev(cb func(lbmap.MaglevOuterKey, lbmap.MaglevOute
 	return f.impl.DumpMaglev(cb)
 }
 
+func (f *FaultyLBMaps) ExistsSockRevNat(cookie uint64, addr net.IP, port uint16) bool {
```

Comment:
Umm... What's a faulty vs fake LB map? How is it relevant to this PR? Please consider splitting this change into a separate commit.

Reply:
FaultyLBMaps is wrapping `LBMaps` to provide fault injection for the tests. Why split into separate commit?